lib/texts.js

```diff
@@ -152,6 +152,8 @@ ${leaves}`,
 	'errorType.syntaxError.wrongBody': template`Body field value is exceeding 4KB limit. ${0}`,
 	'errorType.syntaxError.wrongUrl': template`This does not look like a valid URL. ${0}`,
 	'errorType.syntaxError.modifierMissing': template`${0} chain missing modifier.`,
+	'errorType.suitestCommand': template`provided for 'suitest ${0}' command. It`,
```

Comment:
This message does not seem to be complete. 
Either explain the necessity for this in the comment next to this line, or change code so that error message template makes sense.

Reply:
@AntonSBilan please check this, but looks ok

Reply:
@givehug In this PR I just moved string to texts.js the error messages for automated and interactive arguments are still the same as before the output in case of invalid `username` is `Invalid input provided for 'suitest interactive' command. It should be string`